data/map planets.txt

```diff
@@ -2504,6 +2549,24 @@ planet Luna
 		threshold 7500
 		fleet "Large Republic" 10
 
+planet "Mordente-Bridi"
+	attributes ccor military station urban
+	landscape land/city6
+	music ambient/machinery
+	description `A testament to the engineering corps of the now defunct CCOR, the massive Mordente-Bridi Station is one of the largest in human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest alloy the CCOR could manufacture at scale. Bridi Cylinder is pressurized and contains crew quarters and logistical infrastructure amounting to a significant city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
```

Comment:
```suggestion
	description `A testament to the ingenuity of the engineering corps of the now defunct CCOR, the massive Mordente-Bridi Station is one of the largest in human space. Twin cylinders Mordente and Bridi make up the station's hull, built from the strongest alloy the CCOR could manufacture at scale. Bridi Cylinder is pressurized and contains crew quarters and logistical infrastructure amounting to that of a sizable city, while Mordente Cylinder contains what was the Outer Rim's main shipyard.`
```
Could also use `resourcefulness` or `prowess` instead of `ingenuity` if you think that fits better?